### ISSUE TITLE
Footnotes pound sign is not highlighted on search

### ISSUE THREAD

**Issue by epicfaace** [cestastanford/grandtour#45]
http://grand-tour-explorer-2017.herokuapp.com/#/entries/2803

**Comment by ryanctan** [comment]
What does this mean? Is it still an issue? @epicfaace

**Comment by epicfaace** [comment]
Yes. Do a free search for a term such as "John", then click an entry -- then the term "John" will be highlighted.

However, if you do a free search for the term "#", the similar highlight doesn't take place.

This is a really minor issue though, just something we noticed a while back.
